Pod/Classes/BONText.m

```diff
@@ -305,8 +315,13 @@ - (id)copyWithZone:(NSZone *)zone
     text.string = self.string;
     text.image = self.image;
     text.nextText = self.nextText;
+
+    // !!!: why are you not copying indentSpace too
```

Comment:
#answer `indentSpacer` is backed by `internalIndentSpacer`:

```objc
- (CGFloat)indentSpacer
{
    return self.internalIndentSpacer.doubleValue;
}

- (void)setIndentSpacer:(CGFloat)indentSpacer
{
    self.internalIndentSpacer = @(indentSpacer);
}
```

This is so that we can distinguish between a value of `0.0` and a value of `nil`, which should have different behaviors.